src/LibraryViewExtension/packages.config

```diff
@@ -4,7 +4,7 @@
   <package id="cef.redist.x86" version="3.2987.1601" targetFramework="net452" />
   <package id="CefSharp.Common" version="57.0.0" targetFramework="net45" />
   <package id="CefSharp.Wpf" version="57.0.0" targetFramework="net45" />
-  <package id="Greg" version="1.0.6176.18754" targetFramework="net45" />
+  <package id="Greg" version="1.0.6816.34065" targetFramework="net452" />
```

Comment:
@mjkkirschner any reason this targets 4.5.2 as opposed to 4.5

Reply:
I'm not sure, the package itself targets net45- but the libraryExtension dll itself targets 452 - perhaps that is the reason?